WeakAuras/WeakAuras.lua

```diff
@@ -394,10 +394,10 @@ function WeakAuras.ActivateAuraEnvironment(id, cloneId, state)
         local func = WeakAuras.customActionsFunctions[id]["init"];
         if func then
           current_aura_env.id = id;
+          data.init_started = 1;
```

Comment:
you need to set it regardless of whether there's a action or not, thus do it before the if